tasks/droplet.yml

```diff
@@ -0,0 +1,23 @@
+# digital_ocean module doesn't handle a change in backups_enabled, so do it manually
+- block:
+    - name: Get Droplet information
+      uri:
+        url: https://api.digitalocean.com/v2/droplets/{{ do_id }}
+        headers:
+          Authorization: Bearer {{ lookup('env', 'DO_API_TOKEN') }}
+      register: droplet_info
+      check_mode: false
+    - name: Enable backups on Droplet
+      uri:
+        url: https://api.digitalocean.com/v2/droplets/{{ do_id }}/actions
+        method: POST
+        headers:
+          Authorization: Bearer {{ lookup('env', 'DO_API_TOKEN') }}
+        body_format: json
+        body:
+          type: enable_backups
+        status_code: 201
+      when: "'backups' not in droplet_info.json.droplet.features"
```

Comment:
Interesting - is this just a console message or?

Reply:
No, it's a [conditional](https://docs.ansible.com/ansible/devel/user_guide/playbooks_conditionals.html#the-when-statement). It only enables backups if not previously enabled.